nilearn/plotting/displays.py

```diff
@@ -716,7 +716,9 @@ def add_contours(self, img, threshold=1e-6, filled=False, **kwargs):
             (from matplotlib) for contours and contour_fillings can be
             different.
         """
-        self._map_show(img, type='contour', **kwargs)
+        if not filled:
```

Comment:
Seems a bit unrelated. Is this an obvious fix or does it require some explanations ?

Reply:
It is related and bug raised by @GaelVaroquaux 
https://3567-1235740-gh.circle-artifacts.com/0/home/circleci/project/doc/_build/html/auto_examples/03_connectivity/plot_compare_resting_state_decomposition.html#sphx-glr-auto-examples-03-connectivity-plot-compare-resting-state-decomposition-py

the contours are not filled in some positions.

This fixes that:
https://3645-1235740-gh.circle-artifacts.com/0/home/circleci/project/doc/_build/html/auto_examples/03_connectivity/plot_compare_resting_state_decomposition.html#sphx-glr-auto-examples-03-connectivity-plot-compare-resting-state-decomposition-py